src/scoringreport.rs

```diff
@@ -0,0 +1,16 @@
+use std::fs;
+use crate::{config::Check};
+
+/// Writes to Scoring Report
+pub fn write_to_scoring_report (file: &String, scored_vector: impl Iterator<Item = Check>) {
+  let mut text = String::from("<ul>\n");
```

Comment:
instead of pushing to a string you could have a `Vec<String>` and then [join](https://doc.rust-lang.org/std/primitive.slice.html#method.join) it with `\n`